packages/node_modules/samples/browser-call-with-screenshare/app.js

```diff
@@ -10,67 +10,224 @@
 /* eslint-disable arrow-body-style */
 /* eslint-disable max-len */
 
-// Declare some globals that we'll need throughout
-let activeMeeting, remoteShareStream, webex;
 
-// First, let's wire our form fields up to localStorage so we don't have to
-// retype things everytime we reload the page.
+let redirect_uri;
+let activeMeeting;
+let remoteShareStream;
+let isToken;
+let isOAuth;
+let webex;
+
+function getOAuthRedirectURI() {
+  // This is one way you might programattically determine your redirect_uri
+  // depending on where you've deployed your app, but you're probably better off
+  // having development/staging/production builds and injecting directly from the
+  // environment.
+  redirect_uri = `${window.location.protocol}//${window.location.host}`;
+
+  if (window.location.pathname) {
+    redirect_uri += window.location.pathname;
+  }
+}
 
-[
-  'access-token',
-  'invitee'
-].forEach((id) => {
-  const el = document.getElementById(id);
 
-  el.value = localStorage.getItem(id);
-  el.addEventListener('change', (event) => {
-    localStorage.setItem(id, event.target.value);
+// OAuth section
+if (window.location.hash) {
```

Comment:
Can you move this to the bottom of the file with the call to `handleAuthTypeChange`. I'm afraid this will be missed by consumers trying to use the sample code because it is nested in the function definitions.